src/Domain/Entity/User.php

```diff
@@ -4,42 +4,13 @@
 
 class User
 {
-    private $id;
     private $user_name;
     private $email;
+    public $id;
```

Comment:
I don't like this public here. Is better to add as private, and then create a public method to access this variable. This way, we encapsulate the variable and protect the variable and is only accessible with the getter.